### ISSUE TITLE
TypeError: log is not a function

### ISSUE THREAD

**Issue by jvanoostveen** [pghalliday/mocha-sonar-reporter#9]
I was trying out this reporter, but it looks like there is something wrong or not mentioned in the config.

Command to run it:

```
.\node_modules\.bin\mocha --reporter mocha-sonar-reporter .\build\js\test
```

Output:

```
C:\Users\user\Projects\prj\node_modules\mocha-sonar-reporter\lib\Sonar.js:55
    log(tag('testsuite', {
    ^

TypeError: log is not a function
    at Runner.<anonymous> (C:\Users\user\Projects\prj\node_modules\mocha-sonar-reporter\lib\Sonar.js:55:5)
    at emitNone (events.js:91:20)
    at Runner.emit (events.js:185:7)
    at C:\Users\user\Projects\prj\node_modules\mocha\lib\runner.js:786:12
    at done (C:\Users\user\Projects\prj\node_modules\mocha\lib\runner.js:628:7)
    at next (C:\Users\user\Projects\prj\node_modules\mocha\lib\runner.js:595:16)
    at done (C:\Users\user\Projects\prj\node_modules\mocha\lib\runner.js:628:7)
    at next (C:\Users\user\Projects\prj\node_modules\mocha\lib\runner.js:599:14)
    at Runner.hookErr (C:\Users\user\Projects\prj\node_modules\mocha\lib\runner.js:462:7)
    at Runner.uncaught (C:\Users\user\Projects\prj\node_modules\mocha\lib\runner.js:709:19)
    at process.uncaught (C:\Users\user\Projects\prj\node_modules\mocha\lib\runner.js:778:10)
    at emitOne (events.js:96:13)
    at process.emit (events.js:188:7)
    at process._fatalException (node.js:250:26)
```

Environment:
Windows 10, PowerShell.

When I run the tests with the `--recursive` flag, no error but also no output is shown/generated.
I've added this to `package.json`:

```
  "config": {
    "mocha-sonar-reporter": {
      "outputfile": "build/report/TEST-all.xml"
    }
  },
```

but that makes no difference.

**Comment by pghalliday** [comment]
looks like the mocha interface changed - may even have been a while ago. I have a feeling that the original xunit reporter was fixed (not sure).

I can update this library anyway I guess

**Comment by pghalliday** [comment]
Fixed in version 0.1.6

**Comment by zwik** [comment]
As it is fixed in 0.1.6, I guess the ticket can be closed :) .
